packages/blade/src/components/Alert/__tests__/Alert.web.test.tsx

```diff
@@ -1,5 +1,106 @@
+import userEvent from '@testing-library/user-event';
+
+import { Alert } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import assertAccessible from '~src/_helpers/testing/assertAccessible.web';
+
 describe('<Alert />', () => {
-  it.skip('should render', () => {
-    // todo
+  it('should render', () => {
+    const { container } = renderWithTheme(
+      <Alert description="Currently you can only accept payments in international currencies using PayPal." />,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render positive intent and full width', () => {
+    const { container } = renderWithTheme(
+      <Alert
+        description="Currently you can only accept payments in international currencies using PayPal."
+        intent="positive"
+        isFullWidth
+      />,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render negative intent and borderless', () => {
+    const { container } = renderWithTheme(
+      <Alert
+        description="Currently you can only accept payments in international currencies using PayPal."
+        intent="negative"
+        isBorderless
+      />,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should handle onClick on actions', async () => {
+    const user = userEvent.setup();
+    const onClickPrimary = jest.fn();
+    const onClickSecondary = jest.fn();
+    const { getByText } = renderWithTheme(
+      <Alert
+        description="Currently you can only accept payments in international currencies using PayPal."
+        actions={{
+          primary: { text: 'Primary', onClick: onClickPrimary },
+          secondary: { text: 'Link', onClick: onClickSecondary },
+        }}
+      />,
+    );
+    const primaryAction = getByText('Primary');
+    const secondaryAction = getByText('Link');
+    await user.click(primaryAction);
+    await user.click(secondaryAction);
+
+    expect(onClickPrimary).toHaveBeenCalledTimes(1);
```

Comment:
Do we pass any events or something to onClick? if so let's test it.

Reply:
We just pass the callback to `BaseButton` and `BaseLink` which already have their tests for this. We don't have any usecase for events here though